entity-framework/core/what-is-new/ef-core-3.0/breaking-changes.md

```diff
@@ -356,6 +505,58 @@ This change was made to avoid erroneously defining a primary key property on the
 
 If the property was intended to be the foreign key, and hence part of the primary key, then explicitly configure it as such.
 
+## Database connection is now closed if not used anymore before the TransactionScope has been completed
+
+[Tracking Issue #14218](https://github.com/aspnet/EntityFrameworkCore/issues/14218)
+
+This change will be introduced in EF Core 3.0-preview 4.
+
+**Old behavior**
+
+Before EF Core 3.0, if the context was created with a closed connection it will be maintained opened while the current `TransactionScope` is active.
```

Comment:
A bit hard to read IMO. What does "it" refers to? Does this apply to me if I didn't _pass_ a connection at all? Mixing of past, present and future tenses? Perhaps this is better:

> Before EF Core 3.0, if the context opens the connection under a `TransactionScope`, the connection remains open while the current `TransactionScope` is active.

FWIW, present tense is preferred for Microsoft docs.